### ISSUE TITLE
2FA Support

### ISSUE THREAD

**Issue by Rohukas** [syncloud/platform#469]
It would be really nice if we got 2fa support in syncloud.
I feel as if having 2fa on Nextcloud, for example, is sort of pointless when the main portal does not support 2fa, therefore making the nextcloud 2fa less secure.
Are there any plans for adding 2fa in the future?

**Comment by cyberb** [comment]
Yes as we are using authelia now in a few new apps we can switch to it on the main device login and allow 2fa
